models/reviews.models.js

```diff
@@ -80,3 +80,14 @@ exports.selectComments = (id) => {
       return response.rows;
     });
 };
+
+exports.insertComment = (id, body) => {
+  return db
+    .query(
+      `INSERT INTO comments (review_id, author, body) VALUES ($1, $2, $3) RETURNING *`,
+      [id, body.username, body.body]
+    )
+    .then((response) => {
+      return response.rows;
+    });
+};
```

Comment:
This is looking good, but you will have to add some bits to take care of the situations I mentioned in the comment about the tests.